### PR TITLE
Added a SYD document on fluentd on docker

### PR DESCRIPTION
**Pull request by nico-shishkin**
# What changed

New SYD added: https://deploy-preview-1158--logz-docs.netlify.app/shipping/log-sources/fluentd-docker.html
https://app.logz.io/#/dashboard/send-your-data?tag=all&collection=prometheus-sources&deployPreviewNumber=1158

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/logzio_collections/_log-sources/fluentd-docker.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: Ship container logs with Fluentd on Docker
+logo:
+  logofile: FluentD_Docker.png
+  orientation: vertical
+short-description: Ship container logs with Fluentd on Docker
+data-source: Fluentd on Docker
+shipping-tags:
```
Comment (yotamloe) [inline _source/logzio_collections/_log-sources/fluentd-docker.md:8]: Can you add a reference to our os project, like in this doc:
chrome-capture (5)
[The doc on github](https://github.com/logzio/logz-docs/blob/master/_source/logzio_collections/_metrics-sources/aws-cloudfront.md)

[Github repo](https://github.com/logzio/fluentd-docker-logs)

File: _source/logzio_collections/_log-sources/fluentd-docker.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+3. {% include log-shipping/log-shipping-token.html %}
+4. Replace `<<LOGZIO_INCLUDE_REGEX>>` with the Regex that matches names of containers that you need to ship logs from. The default value for this setting is `.+`.
+5. Replace `<<LOGZIO_PROXY_URI>>` with your proxy URI.
+6. Replace `<<LOGZIO_PROXY_CERT>>` with your proxy certificate.
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours, and then open [Kibana](https://app.logz.io/#/dashboard/kibana/discover?). You can filter for data of type `docker-fluentd` to see the incoming container logs.
+  
+If you still don’t see your data, see [log shipping troubleshooting](https://docs.logz.io/user-guide/log-shipping/log-shipping-troubleshooting.html).
```
Comment (yotamloe) [inline _source/logzio_collections/_log-sources/fluentd-docker.md:115]: I believe it's useful the add the full configurable parameters [list](https://github.com/logzio/fluentd-docker-logs/blob/main/README.md#parameters)

**Review by yotamloe — CHANGES_REQUESTED** [review]
Amazing work Nico 🚀

**Comment by nico-shishkin** [comment]
Hi @yotamloe , thanks for the feedback! I've amended the document - could you please check? Thanks!

**Review by yotamloe — APPROVED** [review]
Amazing work

**Review by ralongit — APPROVED** [review]

**Review by Doron-Bargo — APPROVED** [review]
Great work

**Pull request merged**